src/integration-test/groovy/de/iteratec/osm/measurement/environment/wptserver/PersistingResultsWithCsiIntegrationSpec.groovy

```diff
@@ -63,22 +63,23 @@ class PersistingResultsWithCsiIntegrationSpec extends NonTransactionalIntegratio
     }
 
     void "EventResults of all steps will be saved if some have a customer satisfaction while others have not."() {
-
-        given: ""
-        createTestDataCommonToAllTests()
-        File file = new File("src/test/resources/WptResultXmls/MULTISTEP_1Run_5Steps.xml")
-        WptResultXml xmlResult = new WptResultXml(new XmlSlurper().parse(file))
-        JobResult.build(job: job, expectedSteps: 5, jobConfigRuns: 1, firstViewOnly: true,
-                testId: xmlResult.testId, jobResultStatus: JobResultStatus.RUNNING)
-
-        when: ""
-        jobResultPersisterService.handleWptResult(xmlResult, xmlResult.testId, job)
-        List<EventResult> eventResults = EventResult.list()
-
-        then: ""
-        JobResult.list().size() == 1
-        eventResults.size() == 5
-        eventResults.findAll { it.csByWptDocCompleteInPercent }.size() == 3
+        Job.withNewSession {
```

Comment:
That's again a mistake from the past, but since we are at it, please add some descriptions to given, when and then.